kong/plugins/jwt/schema.lua

```diff
@@ -8,6 +9,13 @@ local function check_user(anonymous)
   return false, "the anonymous user must be empty or a valid uuid"
 end
 
+local function check_positive(v)
+  if v < 0 then
+    return false, "should be >= 0"
```

Comment:
Another convention our error messages follow is to use words instead of signs. In this case:
```
should be 0 or greater
or:
cannot be negative
```